rest/data/host_create.go

```diff
@@ -309,6 +310,7 @@ func makeDockerIntentHost(ctx context.Context, env evergreen.Environment, taskID
 		RegistryName:     createHost.Registry.Name,
 		RegistryUsername: createHost.Registry.Username,
 		RegistryPassword: createHost.Registry.Password,
+		StdinData:        createHost.StdinFileContents,
```

Comment:
Could this be potentially a very very large thing we're now storing in the host document? Should it have some sort of limit? And/or should we unset it once the stream is done?

Reply:
It could theoretically be large, but:

* We already seemingly allow very large unbounded arrays in lots of our data models (like the list of all batchtime tasks in the version document) that can be many MB in size, and they haven't caused problems (it has to be ~5 bibles in length).
* Realistically the input won't be large, because the one user who wanted this said they wanted to pass in a few secrets through stdin instead of having integration with a shared secret store (like Secrets Manager).
* If the input really is that large, their host.create will just fail, and they'll have to come up with their own workaround anyways.

In any case, this one is just a holdover until Docker host.create is deprecated in favor of container tasks, so no matter what is the use case, this is supposedly temporary.

Reply:
~5 Bibles LOL alright that works